lib/ansible/module_utils/api.py

```diff
@@ -114,3 +115,53 @@ def retried(*args, **kwargs):
 
         return retried
     return wrapper
+
+
+def generate_jittered_backoff(retries=10, delay=3, max_delay=60):
+    """The "Full Jitter" backoff strategy described here https://www.awsarchitectureblog.com/2015/03/backoff.html
```

Comment:
Let's make this PEP 257 compliant:
```suggestion
    """The "Full Jitter" backoff strategy.

    Ref: https://www.awsarchitectureblog.com/2015/03/backoff.html
```